README.md

```diff
@@ -44,23 +44,13 @@ Read the [Roadmap](./ROADMAP.md).
 
 ## Quick start
 
-Choose one of the following tutorials:
+Read the [Quick Start Guide](https://pingcap.com/docs/v3.0/tidb-in-kubernetes/tidb-operator-overview/), which includes all the guides for managing TiDB clusters in Kubernetes.
```

Comment:
Suggest changing to "all the quick start guides"

Reply:
copy from tidb: https://github.com/pingcap/tidb#quick-start